src/style-spec/reference/v8.json

```diff
@@ -3345,6 +3355,17 @@
           }
         }
       },
+      "distance": {
+        "doc": "Returns the shortest distance in meters between the evaluated feature and the input geometry. The evaluated feature and input value can be a valid GeoJSON of type `Point`, `MultiPoint`, `LineString`, `MultiLineString`, `Polygon`, `MultiPolygon`, `Feature`, or `FeatureCollection`. Results at zoom level 12 and below may vary in precision due to precision loss in conversions between tile coordinates and `[Longitude, Latitude]` at lower zoom levels.",
```

Comment:
> Results at zoom level 12 and below may vary in precision due to precision loss in conversions between tile coordinates and `[Longitude, Latitude]` at lower zoom levels."

Ah, I could simplify this part to "[encoding geometries](https://docs.mapbox.com/vector-tiles/specification/#encoding-geometry)". Maybe something like:

Distance values returned at zoom level 12 and below may vary in precision due to loss in precision from encoding geometries.

Reply:
@zmiao @asheemmamoowala which do you think is better?

Reply:
> Distance values returned at zoom level 12 and below may vary in precision due to loss in precision from encoding geometries.

☝️